std_msgs/Header header
string mission_file_full_path
